packages/jellyfish-address/src/index.ts

```diff
@@ -0,0 +1,82 @@
+
+import { getNetwork, NetworkName } from '@defichain/jellyfish-network'
+import { Address, AddressType, UnknownTypeAddress } from './Address'
+import { Base58Address } from './Base58Address'
+import { Bech32Address } from './Bech32Address'
+import { P2PKH } from './P2PKH'
+import { P2SH } from './P2SH'
+import { P2WSH } from './P2WSH'
+import { P2WPKH } from './P2WPKH'
```

Comment:
To lowercase all file name and use `_` separator. As this is the standardized approach for this repo.

https://github.com/DeFiCh/jellyfish/blob/main/CONTRIBUTING.md#use-underscores-not-dashes-in-filenames